packages/cypress/src/schematics/cypress-project/cypress-project.ts

```diff
@@ -111,7 +111,7 @@ export default function (options: CypressProjectSchema): Rule {
     }),
     generateFiles(options),
     updateWorkspaceJson(options),
-    updateNxJson(options),
+    options.project ? updateNxJson(options) : noop(),
```

Comment:
Sorry, we do need tags.

Reply:
Okay, I think it's working like you suggested. Let me know if it looks good or if you want some slightly different syntax or something.